tools/RAiDER/gnss/downloadGNSSDelays.py

```diff
@@ -18,7 +18,7 @@
 
 
 # base URL for UNR repository
-_UNR_URL = 'http://geodesy.unr.edu/'
+_UNR_URL = 'https://geodesy.unr.edu/'
```

Comment:
Unrelated to the orbit changes, geodesy.unr.edu is not responding over http today. Requests hang indefinitely, causing the GNSS related tests to fail. The test worked when they last ran on Mar 4.

Requests over https work fine, so updating the scheme here resolved my issues.